corpus/filter.py

```diff
@@ -174,16 +174,18 @@ def run(self):
 
 
 class FilterCorpusBySegmentsJob(Job):
-    def __init__(self, corpus_file, segment_list, compressed=False, invert_match=False):
+    def __init__(
+        self, bliss_corpus, segment_files, compressed=False, invert_match=False
+    ):
         """
-        :param Path corpus_file:
-        :param Path segment_list:
+        :param Path bliss_corpus:
+        :param list[Path]|Path segment_files: a single segment file or a list of segment files
```

Comment:
In `CorpusToTextDictJob` class `segment_file` was used, which only accepted a single segment file. Do we want to change this behavior?

Reply:
We could do this later, actually this is a good point. Maybe it should be singular here (as it takes both), so that if we enable "multiple" inputs at other locations, we do not have to change the name.

Reply:
I am fine with that. seems a bit easier to use singular and allow both. especially if jobs get extended from singular to multiple input